pkg/mdlsub/subscriber_callback.go

```diff
@@ -23,14 +23,19 @@ type SubscriberSettings struct {
 	Output      string                `cfg:"output"`
 	RunnerCount int                   `cfg:"runner_count" default:"10" validate:"min=1"`
 	SourceModel SubscriberSourceModel `cfg:"source"`
-	TargetModel mdl.ModelId           `cfg:"target"`
+	TargetModel SubscriberTargetModel `cfg:"target"`
 }
 
 type SubscriberSourceModel struct {
```

Comment:
Maybe we're just naming this `SubscriberModel` and use it for source and target instead of duplicating it?

Reply:
agreed and fixed